src/test/java/de/retest/recheck/persistence/bin/KryoPersistenceTest.java

```diff
@@ -92,4 +93,26 @@ void on_error_file_should_be_deleted() throws IOException {
 				.isInstanceOf( KryoException.class );
 		assertThat( nonexistent ).doesNotExist();
 	}
+
+	@Test
+	void isCompatible_should_be_compatible_with_equal_or_higher_value() throws Exception {
+		final KryoPersistence<TestReport> cut = new KryoPersistence<>();
```

Comment:
Left-hand side can use the interface `Persistence`.

Reply:
I am using `isCompatible` below which requires the KryoPersistence. This is also in line wit the other methods, which all use `KryoPersistenc`.